docs/source/gbd2017_models/risks/risk_exposure_template.rst

```diff
@@ -1,53 +1,180 @@
-.. _2017_risk_exposure_template:
+.. _2017_exposure_template:
 
-----------------------
+======================
 Risk Exposure Template
-----------------------
+======================
+
+.. important::
+
+   To begin documenting a risk exposure in GBD 2017 for a Vivarium simulation, start by
+   following these steps (after you have :ref:`created a new git branch
+   <contributing>` to work in):
+
+   #. Make a subdirectory :file:`{exposure_name}/` in the folder
+      :file:`gbd2017/risks/` , where :file:`{exposure_name}` is replaced with the
+      (potentially shortened) name of the exposure you are modeling, such as :file:`unsafe_water_source/` or :file:`low_birthweight_short_gestation/`.  This
+      subdirectory is where you will put all the files for your exposure model
+      documentation, including this document, image files, .csv files, etc.

```

Comment:
I'd be in favor of explicitly saying "risk exposure" instead of just "exposure," and using "risk_name" to refer to the subdirectory.  Even though this document is supposed to be focused on just the "exposure" piece of the risk, I think it might be clearer to think of it as a document about the risk as a whole, even though we may have separate documents for the risk effects.

Reply:
Do you think it's overdoing it to go ahead and label everything with `risk_exposure` as opposed to just `risk` or `exposure`?
e.g., Make a subdirectory :file:`{risk_exposure_name}/` and `_2017_risk_exposure_template:`

?

Reply:
Possibly, but I think that might be clearer.